app/src/main/java/com/example/bookwormadventuresdeluxe2/NotificationUtility/APIService.java

```diff
@@ -0,0 +1,20 @@
+package com.example.bookwormadventuresdeluxe2.NotificationUtility;
```

Comment:
Make sure to leave a comment describing what this class is for

Reply:
sorry for the delay. Renamed and added comments to all files.